packages/zowe-explorer-api/src/logger/IZoweLogger.ts

```diff
@@ -30,7 +30,7 @@ type Log4JsCfg = {
 
 const LOGGER_CONFIG: Log4JsCfg = loggerConfig;
 
-export enum MessageSeverityEnum {
+export enum MessageSeverity {
```

Comment:
will this be a possible breaking change for anyone having the `@zowe/zowe-explorer-api` as a dependency?

One way to avoid the braking change is to add a type with the old name
```javascript
/**
 * @deprecated Please use MessageSeverity
 */
export type MessageSeverityEnum = MessageSeverity;
```